packages/core/src/payment/strategies/square/square-script-loader.ts

```diff
@@ -18,17 +18,20 @@ export default class SquareScriptLoader {
     }
 
     async load(testmode?: boolean): Promise<SquareScriptCallBack> {
-        await this._scriptLoader
+        return this._scriptLoader
```

Comment:
Can we try and use async/await here?

Reply:
Done, I moved the code as it was and threw the error from the strategy.